config/community_images.json

```diff
@@ -11,10 +11,10 @@
 {
     "id": "RStudio",
     "label": "RStudio (R 4.1.2, Bioconductor 3.14, Python 3.8.5)",
-    "version": "3.14.0",
-    "updated": "2021-11-09",
+    "version": "3.14.1",
+    "updated": "2022-02-08",
     "packages": "https://storage.googleapis.com/terra-docker-image-documentation/placeholder.json",
-    "image": "us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:3.14.0",
+    "image": "us.gcr.io/broad-dsp-gcr-public/anvil-rstudio-bioconductor:3.14.1-0",
```

Comment:
why do we have `3.14.1-0`?

Reply:
discussed in slack thread https://broadinstitute.slack.com/archives/GB1VAM14J/p1644357701381199?thread_ts=1643811728.672289&cid=GB1VAM14J
going to update now